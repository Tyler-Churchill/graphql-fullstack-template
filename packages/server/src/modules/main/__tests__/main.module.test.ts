import { gql } from 'apollo-server-core';
import TestClient from '../../../tests/TestClient';

it('Server isReady returns true', async () => {
  const resp = await TestClient.query({
    query: gql`
      query {
        isReady
      }
    `
  });

  expect(resp).toMatchSnapshot();
});
