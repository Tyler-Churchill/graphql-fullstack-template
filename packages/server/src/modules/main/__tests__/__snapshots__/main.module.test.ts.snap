// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server isReady returns true 1`] = `
Object {
  "data": Object {
    "isReady": true,
  },
  "loading": false,
  "networkStatus": 7,
  "stale": false,
}
`;
